doc_src/cmds/string-pad.rst

```diff
@@ -0,0 +1,52 @@
+string-pad - pad characters before and after string
+===================================================
+
+Synopsis
+--------
+
+.. BEGIN SYNOPSIS
+
+::
+
+    string pad [(-r | --right)] [(-c | --char) CHAR] [(-w | --width) INTEGER] [STRING...]
+
+.. END SYNOPSIS
+
+Description
+-----------
+
+.. BEGIN DESCRIPTION
+
+``string pad`` pads specified character before and after the string for each STRING.
+
+The default behaviour is left padding with spaces and default width is the length of string (hence, no padding).
```

Comment:
I guess we mostly use American English `behavior`

Reply:
My bad, fixed. 👍